PathwayCommons/app-ui
# cleanup enrichment tests #963

All 12 tests should now pass
- fix parameter names ('p-value' to 'p_value')
- update expected results
- change test names
- change mechanism for checking visualization errors
